### ISSUE TITLE
Do not report issues for commented out code

### ISSUE THREAD

**Issue by rfalke** [jimhester/lintr#821]
Consider the following code:
```
# if(sum(is.na(foo)) == nrow(foo)){
#   die('bar')
# }
```

running with lintr_2.0.1:

```
> lintr::lint("/tmp/lintr-test.R")
/private/tmp/lintr-test.R:1:34: style: There should be a space between right parenthesis and an opening curly brace.
# if(sum(is.na(foo)) == nrow(foo)){
                                 ^~
/private/tmp/lintr-test.R:2:5: style: Commented code should be removed.
#   die('bar')
    ^~~~~~~~~~
```

I'm expected the second error but not the first one. How can I exclude such errors?

**Comment by MichaelChirico** [comment]
I believe that false positive is fixed in the current dev version, can you
confirm whether that's right?

**Comment by AshesITR** [comment]
The current development version produces this output:

```r
lintr::lint("# if(sum(is.na(foo)) == nrow(foo)){
#   die('bar')
# }
")
#> <text>:2:5: style: Commented code should be removed.
#> #   die('bar')
#>     ^~~~~~~~~~
